NEWS.md

```diff
@@ -1,10 +1,19 @@
+# epidatr 1.2.0
```

Comment:
is this 1.2.0 instead of 1.1.3 so that when we do a release all the changes are gathered together?

Reply:
Exactly. Figured the 1.1.1 deviation is fine, since that was a little hotfix release.